features/facebookImport/src/components/messagesMiniStory/messagesMiniStory.jsx

```diff
@@ -2,11 +2,12 @@ import React from "react";
 
 import i18n from "!silly-i18n";
 
+const l12nt = (o) => i18n.l12n.t(o);
+
 import BarChart from "../dataViz/barChart.jsx";
 import InfoButton from "../buttons/infoButton/infoButton.jsx";
 
 import "./messagesMiniStory.css";
-import { formatTime } from "../../utils/formatTime";
```

Comment:
Lazy question: Shouldn't `../../utils/formatTime` be removed now?

Reply:
Ah, right. Not only that, probably elsewhere.

Reply:
... But probably for another PR.